testing/tests/DevExpress.viz.charts/chart.tests.js

```diff
@@ -138,6 +138,35 @@ QUnit.test('Create', function(assert) {
     assert.ok(this.LayoutManager.calledWithNew());
 });
 
+QUnit.test('Chart should have default value of the aggregatebycategory = true', function(assert) {
```

Comment:
let's name the option exactly how it's named in docs?
```suggestion
QUnit.test('Chart should have default value of the aggregateByCategory = true', function(assert) {
```

Reply:
Fixed